<ui version="4.0" stdsetdef="1" >
  <author>Sebastian Trueg</author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>base_K3bAdvancedDataImageSettings</class>
  <widget class="QWidget" name="base_K3bDataCustomFilesystemsWidget" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>552</width>
        <height>320</height>
      </rect>
    </property>
    <property name="windowTitle" >
      <string>Custom Data Filesystems</string>
    </property>
    <layout class="QGridLayout" >
      <property name="margin" >
        <number>0</number>
      </property>
      <property name="spacing" >
        <number>6</number>
      </property>
      <item rowspan="1" row="1" column="0" colspan="2" >
        <widget class="Q3GroupBox" name="groupIsoSettings" >
          <property name="title" >
            <string>File System Settings</string>
          </property>
          <layout class="QVBoxLayout" >
            <item>
              <widget class="K3ListView" name="m_viewIsoSettings" >
                <property name="fullWidth" >
                  <bool>true</bool>
                </property>
                <column>
                  <property name="text" >
                    <string>ISO9660 Filesystem</string>
                  </property>
                  <property name="clickable" >
                    <bool>false</bool>
                  </property>
                  <property name="resizable" >
                    <bool>false</bool>
                  </property>
                </column>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item row="0" column="0" >
        <widget class="Q3GroupBox" name="groupBox3" >
          <property name="title" >
            <string>File Systems</string>
          </property>
          <layout class="QVBoxLayout" >
            <item>
              <widget class="QCheckBox" name="m_checkRockRidge" >
                <property name="text" >
                  <string>&amp;Generate Rock Ridge extensions</string>
                </property>
                <property name="checked" >
                  <bool>true</bool>
                </property>
                <property name="toolTip" stdset="0" >
                  <string>Add Rock Ridge extensions to the file system</string>
                </property>
                <property name="whatsThis" stdset="0" >
                  <string>&lt;p>If this option is checked, K3b will generate the System Use Sharing Protocol records (SUSP) specified by the Rock Ridge Interchange Protocol (IEEE-P1282).
&lt;p>Rock Ridge extends the ISO-9660 filesystem by features equal to the UNIX filesystems (permissions, symbolic links, very long filenames, ...). It uses ISO-8859 or UTF-16 based characters and allows 255 octets.
&lt;p>Rock Ridge extensions are located at the end of each ISO-9660 directory record. This makes the Rock Ridge tree closely coupled to the ISO-9660 tree.
&lt;p>&lt;b>It is highly recommended to use Rock Ridge extensions on every data CD or DVD.&lt;/b></string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QCheckBox" name="m_checkJoliet" >
                <property name="text" >
                  <string>Generate &amp;Joliet extensions</string>
                </property>
                <property name="toolTip" stdset="0" >
                  <string>Add Joliet extensions to the file system</string>
                </property>
                <property name="whatsThis" stdset="0" >
                  <string>&lt;p>If this option is checked, K3b will add additional Joliet extensions to the ISO-9660 file system.
&lt;p>Joliet is not an accepted independent international standard like ISO-9660 or Rock Ridge. It is mainly used on Windows systems.
&lt;p>Joliet does not allow all characters, so the Joliet filenames are not identical to the filenames on disk (as compared to Rock Ridge). Joliet has a filename length limitation of 64 chars (independent from the character coding and type e.g. European vs. Japanese). This is inconvenient, as modern file systems all allow 255 characters per path name component.
&lt;p>Joliet uses UTF-16 coding.
&lt;p>&lt;b>Caution:&lt;/b> With the exception of Linux and FreeBSD, there is no POSIX-like OS that supports Joliet. So &lt;b>never create Joliet-only CDs or DVDs&lt;/b> for that reason.</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QCheckBox" name="m_checkUdf" >
                <property name="text" >
                  <string>Generate &amp;UDF structures</string>
                </property>
                <property name="toolTip" stdset="0" >
                  <string>Add UDF structures to the file system</string>
                </property>
                <property name="whatsThis" stdset="0" >
                  <string>&lt;p>If this option is checked K3b will create UDF filesystem structures in addition to the ISO9660 filesystem.
&lt;p>The UDF (&lt;em>&lt;b>U&lt;/b>niversal &lt;b>D&lt;/b>isk &lt;b>F&lt;/b>ormat&lt;/em>) is mainly used for DVDs.</string>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item row="0" column="1" >
        <widget class="Q3GroupBox" name="groupBox5" >
          <property name="title" >
            <string>Other Settings</string>
          </property>
          <layout class="QVBoxLayout" >
            <item>
              <widget class="QCheckBox" name="m_checkPreservePermissions" >
                <property name="text" >
                  <string>Preserve file permissions (bac&amp;kup)</string>
                </property>
                <property name="whatsThis" stdset="0" >
                  <string>&lt;p>If this option is checked, all files in the resulting file system will have exactly the same permissions as the source files. (Otherwise, all files will have equal permissions and be owned by root).
&lt;p>This is mainly useful for backups.&lt;p>&lt;b>Caution:&lt;/b> The permissions may not make much sense on other file systems; for example, if a user that owns a file on the CD or DVD does not exist.</string>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
    <customwidgets>
    <customwidget>
      <class>K3ListView</class>
      <extends>Q3ListView</extends>
      <header location="global" >k3listview.h</header>
    </customwidget>
  </customwidgets>
  <tabstops>
    <tabstop>m_checkRockRidge</tabstop>
    <tabstop>m_checkJoliet</tabstop>
  </tabstops>
  <includes>
    <include location="local" >k3listview.h</include>
  </includes>
</ui>
